docs/_templates/auto_api_templates/python/module.rst

```diff
@@ -3,26 +3,88 @@
 
 {% endif %}
 
+:mod:`{{ obj.name }}`
+======={{ "=" * obj.name|length }}
+
 {% if 'neptune' == obj.name %}
 
+.. image:: ../../../../_static/images/api_references/Neptune_Object_Hierarchy_V1.jpg
```

Comment:
I like this image and this section but I think we could add a paragraph explaining this global neptune object by saying

neptune.init()

connects to neptune and sets a global project context

note -> you can also set it explicitly read projects

neptune.craete_experiment()

creates and sets experiment object in the current global project context. 

note -> you can also set it explicitly read experiments

I think this short paragraph can make things even clearer for the reader.